app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Articles;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class PostController extends Controller
+{
+    public function index()
```

Comment:
Remember to follow op on these automatically generated PHPcs comments. In the exam, there might be a point deduction when we find them in your code.